src/stores/ui.js

```diff
@@ -0,0 +1,8 @@
+import { defineStore } from 'pinia'
+export const useUiStore = defineStore('ui', () => {
```

Comment:
I still prefer `ui store` over `app store` because it specifically targets UI-related activity. AppStore sounds too generic to me and suggests there might be also some business logic.